util/src/main/java/io/kubernetes/client/Exec.java

```diff
@@ -349,7 +349,16 @@ static int parseExitCode(ApiClient client, InputStream inputStream) {
     return -1;
   }
 
-  protected static class ExecProcess extends Process {
+  public interface ResizableStream {
+    OutputStream getResizeStream();
```

Comment:
Add documentation as to what this is so that people can implement the interface.

Reply:
added documents